### DISCUSSION TITLE
Entity last update time as an optional text

### DISCUSSION THREAD

**Discussion by jose1711** [Clooos/Bubble-Card#947] [Questions about config, custom styles and templates] [answered]
As per README it should be possible to display last update time inside a button. I don't see such option inside GUI and adding it manually in YAML editor makes no difference. What I am doing wrong?

**Comment by Clooos** [comment]
Hi, it is here in the editor (the one at the bottom):

<img width="801" alt="image" src="https://github.com/user-attachments/assets/ad25dfc6-8600-4020-abf2-7ccd79c5b8bf">

**Reply by jose1711** [reply]
I am talking about the last update, not the last change. For instance:
```
{{ states.person.jose.last_updated }} => 2024-11-21 17:08:36.387397+00:00
{{ states.person.jose.last_changed }} => 2024-11-21 17:02:50.145424+00:00
```

**Reply by Clooos** [reply, marked as answer]
Indeed this is not currently possible, but I will add this possibility. I've added this in my to-do list 👌
